### PR TITLE
Feat/cardano token changes

### PR DESCRIPTION
**Pull request by Prashanthsoordelu**
### 📝 Description

Fixed memo bug in pending transactions, error message updates, transaction summary updates

### ❓ Context

- **Impacted projects**: `ledger-live-common` `ledger-live-desktop` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/src/families/cardano/buildSubAccounts.ts [hunk 1]
```diff
@@ -102,9 +106,7 @@ const mapTxToTokenAccountOperation = ({
         ),
         blockHeight: tx.blockHeight,
         date: new Date(tx.timestamp),
-        extra: {
-          memo,
-        },
+        extra: extra,
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/cardano/buildSubAccounts.ts:109]: Cleaner syntax:
`extra: memo ?? {}`

File: libs/ledger-live-common/src/families/cardano/js-synchronisation.ts [hunk 2]
```diff
@@ -68,9 +72,7 @@ function mapTxToAccountOperation(
     subOperations,
     blockHeight: tx.blockHeight,
     date: new Date(tx.timestamp),
-    extra: {
-      memo,
-    },
+    extra: extra,
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/cardano/js-synchronisation.ts:75]: Same here.

File: libs/ledger-live-common/src/families/cardano/js-signOperation.ts [hunk 3]
```diff
@@ -103,9 +107,7 @@ const buildOptimisticOperation = (
     recipients: transaction.getOutputs().map((o) => o.address.getBech32()),
     accountId: account.id,
     date: new Date(),
-    extra: {
-      memo,
-    },
+    extra: extra,
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/cardano/js-signOperation.ts:110]: Same here.

**Review by haammar-ledger — APPROVED** [review]

File: libs/ledger-live-common/src/families/cardano/buildSubAccounts.ts (hunk 1 above)
Comment (gre) [reply libs/ledger-live-common/src/families/cardano/buildSubAccounts.ts:109]: doesn't seem equivalent , it's more like `extra: memo ? {memo} : {}` 🤔

**Pull request merged**